backends/cadence/reference/operators/quantized_conv_out.cpp

```diff
@@ -1,261 +0,0 @@
-/*
- * Copyright (c) Meta Platforms, Inc. and affiliates.
- * All rights reserved.
- *
- * This source code is licensed under the BSD-style license found in the
- * LICENSE file in the root directory of this source tree.
- */
-
-#include <executorch/backends/cadence/reference/kernels/kernels.h>
```

Comment:
can we put the `convolution.cpp` stuff in this one instead, and remove the non-quant version? It should always be quantized anyway, and it will be less confusing